src/ctia/entity/sighting/schemas.clj

```diff
@@ -78,4 +78,5 @@
 
 (def sighting-sort-fields
   (conj sighting-fields
-        "observed_time.start_time,timestamp"))
+        :observed_time.start_time
+        :timestamp))
```

Comment:
This sort field is intentionally exposed as `"observed_time.start_time,timestamp"` for CTR which needs to sort sigthings first on `observed_time.start_time` then `timestamp`. I do not have the full context, but any sort option exposed like this is a requirement of an API client (SecureX, CTR, IROH, Engine, Ribbon, and still counting ;-) ).